mita/async.py

```diff
@@ -21,6 +21,9 @@ def bootstrap_pecan(signal, sender):
         config_path = os.path.abspath(os.path.join(here, '../config/config.py'))
 
     pecan.configuration.set_config(config_path, overwrite=True)
+    # Once configuration is set we need to initialize the models so that we can connect
+    # to the DB wth a configured mapper.
+    models.init_model()
```

Comment:
from testing this it didn't look like it. This was all it took for everything to work properly